addons/website/static/src/scss/website.scss

```diff
@@ -1353,6 +1353,15 @@ header {
                     }
```

Comment:
Commit message:
- `Click on the header and choose "Over The content"...` => nitpick: "Over The Content" (with a capital "C") :wink: 
- `Use the color picker to set the "Text Color" option to "white".` => nitpick so you can keep it like that: to make it simpler, you could just have said `Set the "Text Color" option to a white/really light color` (no need to say "use the colorpicker", but again as you prefer :+1: )  
- `a "dark" background` => nitpick: no need for the quotes around dark (same for white above if you keep what you wrote) :slightly_smiling_face: 
- `button in the "over the content"` => "Over The Content"